Makefile

```
@@ -179,11 +179,11 @@ vet: ## Run go vet against code.
 
 .PHONY: ci-lint
 ci-lint: golangci-lint
-	$(GOLANGCI_LINT) run --timeout 15m0s
+	find . -path ./site -prune -false -o -name go.mod -execdir $(GOLANGCI_LINT) run $(GOLANGCI_LINT_FIX) --timeout 15m0s --config "$(PROJECT_DIR)/.golangci.yaml" \;
```

```suggestion
	find . -path ./site -prune -false -o -name go.mod -execdir $(GOLANGCI_LINT) run $(GOLANGCI_LINT_FIX) --timeout 15m0s --config "$(PROJECT_DIR)/.golangci.yaml";
```

/hold

@mbobrovskyi do we need this one?

---

Yes, this is required for `-execdir` to escape `;`.Without it we have an error:

```
find: -execdir: no terminating ";" or "+"
```

---

Oh, I see. Thank you.
/hold cancel